aspnetcore/security/authorization/secure-data.md

```diff
@@ -135,6 +135,8 @@ The `SeedData` class creates two accounts: administrator and manager. Use the [S
 dotnet user-secrets set SeedUserPW <PW>
 ```
 
+If you don't use a strong password, you'll get an exception when `SeedData.Initialize` is called.
```

Comment:
To avoid the "will" of "you'll," it could go "... password, an exception is thrown at runtime when ..."